crates/bevy_render/src/render_graph/nodes/compute_node.rs

```diff
@@ -0,0 +1,100 @@
+use crate::{
+    dispatch::{ComputeCommand, Dispatch},
+    pipeline::ComputePipelineDescriptor,
+    render_graph::{Node, ResourceSlots},
+    renderer::{BindGroupId, RenderContext},
+};
+use bevy_asset::{Assets, Handle};
+use bevy_ecs::{Resources, World};
+
+pub struct ComputeNode;
+
+impl ComputeNode {
+    pub fn new() -> Self {
+        ComputeNode {}
+    }
+}
+
+impl Default for ComputeNode {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Node for ComputeNode {
+    fn update(
+        &mut self,
+        world: &World,
+        resources: &Resources,
+        render_context: &mut dyn RenderContext,
+        _input: &ResourceSlots,
+        _output: &mut ResourceSlots,
+    ) {
+        let pipelines = resources
+            .get::<Assets<ComputePipelineDescriptor>>()
+            .unwrap();
+        render_context.begin_compute_pass(&mut |compute_pass| {
+            let mut compute_state = ComputeState::default();
+
+            let mut entities = world.query::<&Dispatch>();
```

Comment:
nit: for short queries i generally like inlining them in the for loop. feels "clearer" to me